cmd/controller-manager/app/options/options.go

```diff
@@ -47,6 +54,9 @@ type ServerOption struct {
 	CaCertData           []byte
 	EnableLeaderElection bool
```

Comment:
Leader election related parameters can be placed in an independent flagset, please refer to kubernets to make it more graceful.

Reply:
@Monokaix updated!  pls  check again.